src/redisassert.c

```diff
@@ -46,8 +47,13 @@ void _serverAssert(const char *estr, const char *file, int line) {
 }
 
 void _serverPanic(const char *file, int line, const char *msg, ...) {
+    va_list ap;
+    va_start(ap,msg);
+    char fmtmsg[256];
+    vsnprintf(fmtmsg,sizeof(fmtmsg),msg,ap);
+    va_end(ap);
     fprintf(stderr, "------------------------------------------------");
```

Comment:
```suggestion
    va_list ap;
    char fmtmsg[256];
    
    va_start(ap,msg);
    vsnprintf(fmtmsg,sizeof(fmtmsg),msg,ap);
    va_end(ap);
    
    fprintf(stderr, "------------------------------------------------");
```

Reply:
It looks better that way. Thanks for the suggestion